OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,25 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir
+		{
+			get
+			{
+				var gameDir = AppDomain.CurrentDomain.BaseDirectory;
+
+				// BUG: In Game.cs we get GameDir with trailing DirectorySeparator, but not in PlatformTest.cs
+				if (!gameDir.EndsWith(Path.DirectorySeparatorChar.ToString()))
+				{
+					gameDir += Path.DirectorySeparatorChar;
+				}
```

Comment:
Alright, I'll fix.
make check was happy however.

Enter command: check
Checking for explicit interface violations...
Checking for code style violations in OpenRA.Platforms.Default...
No violations encountered in OpenRA.Platforms.Default.
Checking for code style violations in OpenRA.Game...
No violations encountered in OpenRA.Game.
Checking for code style violations in OpenRA.Mods.Common...
No violations encountered in OpenRA.Mods.Common.
Checking for code style violations in OpenRA.Mods.Cnc...
No violations encountered in OpenRA.Mods.Cnc.
Checking for code style violations in OpenRA.Mods.D2k...
No violations encountered in OpenRA.Mods.D2k.
Checking for code style violations in OpenRA.Utility...
No violations encountered in OpenRA.Utility.
Checking for code style violations in OpenRA.Test...
No violations encountered in OpenRA.Test.
Press enter to continue.